platform_main_feature.strongarm.inc

```diff
@@ -11,6 +11,15 @@
 function platform_main_feature_strongarm() {
   $export = array();
 
+  $strongarm = new stdClass();
+  $strongarm->disabled = FALSE; /* Edit this to true to make a default strongarm disabled initially */
+  $strongarm->api_version = 1;
+  $strongarm->name = 'addtoany_additional_js';
+  $strongarm->value = 'a2a_config.exclude_services = [\'yahoo\', \'telegram\', \'balatarin\', \'blinklist\', \'pocket\', \'aol\', \'myspace\', \'linkedin\', \'whatsapp\', \'pinterest\', \'reddit\', \'mix\',\'tumblr\', \'email\', \'gmail\'];
+a2a_config.num_services = 2;
+a2a_config.prioritize = [\'facebook\', \'twitter\'];';
```

Comment:
Not super familiar with Add To Any, but are these two lines fine? I don't seem to get any errors, but line 20 looks a little funky with the double semi-colon.

Reply:
addtoany has a section for "Additional Javascript". It takes the value and adds it the the script on the page https://www.addtoany.com/buttons/customize/remove_services
The config is a textarea so this gets spit out on separate lines as such (with semi colons):
```javascript
a2a_config.num_services = 2;
a2a_config.prioritize = [\'facebook\', \'twitter\'];
```